### ISSUE TITLE
[MU4 Issue] Long tooltips in palettes don’t break

### ISSUE THREAD

**Issue by akulisch** [musescore/MuseScore#12774]
**Describe the bug**
Longer tooltips of symbols in the palettes don’t have enough space to be fully within their box. 

**To Reproduce**
1. Switch to German (for the examples below; same probably also happens in other languages; maybe even English)
2. Hover mouse over (e.g.) “25 small diesis up, (25S, ~5:13S, ~37S, 5C plus 5C), 2° up [53 EDO]” or add the Accordion palette and hover over (e.g.) Right hand, 3 ranks, 4' stop + lower tremolo 8' stop + 8' stop + upper tremolo 8' stop + 16' stop

**Expected behavior**
Either the tooltip box needs its maximum width removed, or, preferably, line breaks have to be inserted.

**Screenshots**
![image](https://user-images.githubusercontent.com/15906252/184038832-1707e6de-0116-40d3-aa8c-dcd967b767dd.png)
![image](https://user-images.githubusercontent.com/15906252/184038981-bbc4b7e7-056a-4be9-8a63-d53dd047c4fc.png)
![image](https://user-images.githubusercontent.com/15906252/184039063-87382362-f0ef-42cb-9eed-ca98bd5359f1.png)

**Platform information**
- OS: Windows 10 Pro 10.0.19044 Build 19044
- MuseScore Version: v4.0_alpha2

**Comment by Tantacrul** [comment]
Yes, this is super annoying. It would be great to simply have a character limit per line and then bump it down to a second line when needed.

**Comment by abariska** [comment]
FYI @Tantacrul and all. 
Comment from @cbjeukendrup:

> I fixed it for QML tooltips in general, so it should work for all of them, in Palettes, Inspector, Note Input Toolbar…
> However, the master palette is implemented in Qt Widgets. It uses Qt’s tooltip implementation, so it might be different from other tooltips, but we can’t influence that unfortunately.

So, there's no word cutting for now in Master palette but long words still don't wrap. Could be fixed in future

<img width="1025" alt="CleanShot 2022-09-21 at 16 41 56" src="https://user-images.githubusercontent.com/101250347/191521834-7841f364-2fc8-4ee7-a88e-b1ef40e97ea6.png">

**Comment by abariska** [comment]
🟢 **FIXED** in https://github.com/musescore/MuseScore/pull/12801
